services/forwarding/forwarding.rst

```diff
@@ -0,0 +1,90 @@
+.. _doc_mailforwarding:
+.. _Dashboard: https://mailsac.com/dashboard
+.. _`Manage Email Addresses`: http://mailsac.com/addresses
+.. _`Manage Domains`: http://mailsac.com/domains
+
+Email Forwarding
+================
+
+Forwarding allows email to be sent to a different email address or
+service (Webhook, Websocket, Slack Webhook). Private addresses and custom
```

Comment:
Nit: capitalization seems to be:
<img width="647" alt="image" src="https://user-images.githubusercontent.com/1668190/84815752-573a9b00-afc8-11ea-9218-874fa51830af.png">